content/en/blog/2023/native-sidecars/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Kubernetes Native Sidecars in Istio
+description: Demoing the new SidecarContainers feature with Istio.
+publishdate: 2023-07-25
+attribution: "John Howard (Google)"
+keywords: [istio,sidecars,kubernetes]
+---
+
+If you have heard anything about service meshes, it is that they work using the sidecar pattern: a proxy server is deployed alongside your application code.
+The sidecar pattern is just that: a pattern.
+Up until this point, there has been no formal support for 'sidecar containers' in Kubernetes at all.
```

Comment:
I'm not sure on the single quotes and how they look. It doesn't make sense to backtick them. Maybe just remove them.